api/src/main/java/org/openmrs/module/appointments/service/impl/TeleconsultationAppointmentService.java

```diff
@@ -1,14 +1,34 @@
 package org.openmrs.module.appointments.service.impl;
 
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Patient;
+import org.openmrs.PatientIdentifier;
+import org.openmrs.api.PatientService;
 import org.openmrs.api.context.Context;
+import org.openmrs.module.appointments.model.AdhocTeleconsultationResponse;
 import org.openmrs.module.appointments.model.Appointment;
+import org.openmrs.module.appointments.notification.NotificationResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
 
 import java.text.MessageFormat;
+import java.util.List;
+import java.util.Random;
 
+@Component
```

Comment:
We don't need to add @component annotation here. the service is declared in the "moduleApplicationContext.xml". Be careful of component scanning.  Lots of OpenMRS wiring do not work with annotation driven config for services .. like transaction management across services. By default, use annotation for web or other non-service, non-dao, non-transactional components